src/test/java/com/conveyal/gtfs/dto/PatternStopWithFlexDTO.java

```diff
@@ -0,0 +1,83 @@
+package com.conveyal.gtfs.dto;
+
+public class PatternStopWithFlexDTO extends PatternStopDTO {
+
+    // Flex additions.
+    public String location_group_id;
+    public String location_id;
+    public Integer start_pickup_drop_off_window;
+    public Integer end_pickup_drop_off_window;
+    public String pickup_booking_rule_id;
+    public String drop_off_booking_rule_id;
+
+    /** Empty constructor for deserialization */
+    public PatternStopWithFlexDTO() {}
+
+    public PatternStopWithFlexDTO(String patternId, String stopId, int defaultTravelTime, int stopSequence) {
+        pattern_id = patternId;
+        stop_id = stopId;
+        stop_sequence = stopSequence;
+        default_travel_time = defaultTravelTime;
+    }
+
+    public PatternStopWithFlexDTO(String patternId, String stopId, int stopSequence) {
+        pattern_id = patternId;
+        stop_id = stopId;
+        stop_sequence = stopSequence;
+    }
+    public PatternStopWithFlexDTO(String patternId, String stopId, int stopSequence, int defaultTravelTime, int defaultDwellTime) {
+        pattern_id = patternId;
```

Comment:
Reuse the first ctor after the trivial one.

Reply:
Fixed in: https://github.com/ibi-group/gtfs-lib/pull/7/commits/2ce3840e48a92b3dd5a9e85be710e3e56d645c2d